shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/resources/config/config-calcite.yaml

```diff
@@ -23,10 +23,54 @@ rules:
         tableStrategy:
           standard:
             shardingColumn: item_id
-            shardingAlgorithmName: table_inline
+            shardingAlgorithmName: table_inline_item_id
+      t_user_encrypt_calcite_sharding:
+        actualDataNodes: calcite_jdbc_1.t_user_encrypt_calcite_sharding_${0..1}
```

Comment:
> calcite_jdbc_1.t_user_encrypt_calcite_sharding_${0..1}

It is supposed to be `calcite_ds.t_user_encrypt_calcite_sharding_${0..1}` to leverage `replicaQuery` power.